pkgs/tools/backup/mastodon-archive/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, python
```

```suggestion
, python3
```
otherwise it's harder to tell from this file what `python` actually is.

---

Done